### PR TITLE
Fix license headers

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

File: .licenserc.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - '.woke.yaml'
+    - '.woke.yml'
+    - 'CODEOWNERS'
+    - 'icon.svg'
+    - 'LICENSE'
+    - 'trivy.yaml'
+    - 'zap_rules.tsv'
+    - 'lib/**'
+    - 'indico_rock/etc/logging.yaml'
+    - 'indico_rock/etc/uwsgi.ini'
```
Comment (gtrkiller) [inline .licenserc.yaml:37]: We do not have any indico_rock files in this charm.

Comment (arturo-seijas) [reply .licenserc.yaml:37]: Ups. Thx

**Review by nrobinaubertin — APPROVED** [review]

File: .licenserc.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    - '.gitignore'
+    - '.licenserc.yaml'
+    - '.trivyignore'
+    - '.woke.yaml'
+    - '.woke.yml'
+    - 'CODEOWNERS'
+    - 'icon.svg'
+    - 'LICENSE'
+    - 'trivy.yaml'
+    - 'zap_rules.tsv'
```
Comment (gtrkiller) [inline .licenserc.yaml:34]: are the zap_rules and trivy files necessary?

Comment (arturo-seijas) [reply .licenserc.yaml:34]: It's from the default config

**Review by gtrkiller — APPROVED** [review]
LGTM

**Pull request merged**